ansible/roles/virtualbmc/README.md

```diff
@@ -0,0 +1,25 @@
+Virtual BMC
+===========
+
+This role sets up Virtual BMC. It will configure the Virtual BMC daemon in
+systemd, and add the specified domains to the daemon.
```

Comment:
This role is doing host config stuff and per-node stuff. It could be split at some point.

Reply:
Good idea.